### PR TITLE
chore: add release 8.4 post

### PR DESCRIPTION
**Pull request by Trial97**

### CODE REVIEW

File: src/news/release-8.4.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+macos_file_extension: zip
+macos_signature: u4nVbCdMLaPkIYma3VBONC9Nxwb+Nsad4fBqJ8Un9HcBCYSrRjrsja7A+cdWO/mrP8JUt6EyhGgLpGYDT2vZAA==
+tags:
+  - release
+---
+
+Hello there!
+
+We have accumulated a few changes and fixes in the past few weeks.
+The latest PrismLauncher update brings enhanced support for NeoForge and ATLauncher modpacks, along with significant improvements to user experience on macOS and critical bug fixes, including better CurseForge export and stability enhancements. Enjoy the new features and smoother operations with this release!
```
Comment (DioEgizio) [inline src/news/release-8.4.md:16]:
```suggestion
The latest Prism Launcher update brings enhanced support for NeoForge and ATLauncher modpacks, along with significant improvements to user experience on macOS and critical bug fixes, including better CurseForge export and stability improvements. Enjoy the new features and smoother operations with this release!
```

File: src/news/release-8.4.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+Until next time!
+
+## Added
+
+- Add neoforge support for technic packs and atlauncher packs by [@Trial97](https://github.com/Trial97) in [#2382](https://github.com/PrismLauncher/PrismLauncher/pull/2382)
```
Comment (DioEgizio) [inline src/news/release-8.4.md:22]:
```suggestion
- Add NeoForge support for Technic packs and ATLauncher packs by [@Trial97](https://github.com/Trial97) in [#2382](https://github.com/PrismLauncher/PrismLauncher/pull/2382)
```

File: src/news/release-8.4.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+- Add support for bcachefs for Copy-on-Write instance copies by [@KernelFreeze](https://github.com/KernelFreeze) in [#2438](https://github.com/PrismLauncher/PrismLauncher/pull/2438)
+- [macos] Add sdkman to java search paths  by [@dqfan2012](https://github.com/dqfan2012) in [#2377](https://github.com/PrismLauncher/PrismLauncher/pull/2377)
```
Comment (DioEgizio) [inline src/news/release-8.4.md:24]:
```suggestion
- [macOS] Add sdkman to java search paths  by [@dqfan2012](https://github.com/dqfan2012) in [#2377](https://github.com/PrismLauncher/PrismLauncher/pull/2377)
```

File: src/news/release-8.4.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+We have accumulated a few changes and fixes in the past few weeks.
+The latest PrismLauncher update brings enhanced support for NeoForge and ATLauncher modpacks, along with significant improvements to user experience on macOS and critical bug fixes, including better CurseForge export and stability enhancements. Enjoy the new features and smoother operations with this release!
+
+Until next time!
+
+## Added
+
+- Add neoforge support for technic packs and atlauncher packs by [@Trial97](https://github.com/Trial97) in [#2382](https://github.com/PrismLauncher/PrismLauncher/pull/2382)
+- Add support for bcachefs for Copy-on-Write instance copies by [@KernelFreeze](https://github.com/KernelFreeze) in [#2438](https://github.com/PrismLauncher/PrismLauncher/pull/2438)
```
Comment (DioEgizio) [inline src/news/release-8.4.md:23]:
```suggestion
- [Linux] Add support for bcachefs for Copy-on-Write instance copies by [@KernelFreeze](https://github.com/KernelFreeze) in [#2438](https://github.com/PrismLauncher/PrismLauncher/pull/2438)
```

File: src/news/release-8.4.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+- [macos] Add sdkman to java search paths  by [@dqfan2012](https://github.com/dqfan2012) in [#2377](https://github.com/PrismLauncher/PrismLauncher/pull/2377)
+
+## Changed
+
+- Update auth server ping with new domain by [@Trial97](https://github.com/Trial97) in [#2471](https://github.com/PrismLauncher/PrismLauncher/pull/2471)
```
Comment (DioEgizio) [inline src/news/release-8.4.md:28]:
```suggestion
- Update the auth server domain to the correct one to use for pinging in logs by [@Trial97](https://github.com/Trial97) in [#2471](https://github.com/PrismLauncher/PrismLauncher/pull/2471)
```

File: src/news/release-8.4.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+- [linux] shorten desktop entry comment field by [@deadmeu](https://github.com/deadmeu) in [#1032](https://github.com/PrismLauncher/PrismLauncher/pull/1032)
```
Comment (DioEgizio) [inline src/news/release-8.4.md:29]:
```suggestion
- [Linux] Shorten desktop entry comment field by [@deadmeu](https://github.com/deadmeu) in [#1032](https://github.com/PrismLauncher/PrismLauncher/pull/1032)
```

File: src/news/release-8.4.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+- [macos] Improve blocked mods dialog by [@kthchew](https://github.com/kthchew) in [#2379](https://github.com/PrismLauncher/PrismLauncher/pull/2379)
```
Comment (DioEgizio) [inline src/news/release-8.4.md:30]:
```suggestion
- [macOS] Improve blocked mods dialog by [@kthchew](https://github.com/kthchew) in [#2379](https://github.com/PrismLauncher/PrismLauncher/pull/2379)
```

File: src/news/release-8.4.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+- improve invalid character removal from path by [@Trial97](https://github.com/Trial97) in [#2425](https://github.com/PrismLauncher/PrismLauncher/pull/2425) and [#2531](https://github.com/PrismLauncher/PrismLauncher/pull/2531)
```
Comment (DioEgizio) [inline src/news/release-8.4.md:31]: this definitely needs better naming

File: src/news/release-8.4.md [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+
+## Fixed
+
+- Ensure markdown images have a line break when following a list by [@SabrePenguin](https://github.com/SabrePenguin) and [@TheKodeToad](https://github.com/TheKodeToad) in [#2346](https://github.com/PrismLauncher/PrismLauncher/pull/2346)
+- Fix CurseForge export caused by utf8 enabled by [@Trial97](https://github.com/Trial97) in [#2353](https://github.com/PrismLauncher/PrismLauncher/pull/2353)
```
Comment (DioEgizio) [inline src/news/release-8.4.md:36]:
```suggestion
- Fix CurseForge export caused by UTF-8 issues by [@Trial97](https://github.com/Trial97) in [#2353](https://github.com/PrismLauncher/PrismLauncher/pull/2353)
```

File: src/news/release-8.4.md [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+- Fix importing of vanilla technic modpacks by [@Trial97](https://github.com/Trial97) in [#2439](https://github.com/PrismLauncher/PrismLauncher/pull/2439)
+- Fix incorrect launch time by [@Trial97](https://github.com/Trial97) in [#2423](https://github.com/PrismLauncher/PrismLauncher/pull/2423)
+- Fix portable if manifest is missing by [@Trial97](https://github.com/Trial97) in [#2405](https://github.com/PrismLauncher/PrismLauncher/pull/2405)
+- Fix resizing too small causing an index out of range crash by [@coolguy1842](https://github.com/coolguy1842) in [#2474](https://github.com/PrismLauncher/PrismLauncher/pull/2474)
+- Fix: update datapack and resourcepack versions by [@DioEgizio](https://github.com/DioEgizio) in [#2534](https://github.com/PrismLauncher/PrismLauncher/pull/2534)
+- Made Custom New Instance scrollable by [@Trial97](https://github.com/Trial97) in [#2383](https://github.com/PrismLauncher/PrismLauncher/pull/2383)
+- Update TFB App import instance path detection by [@Trial97](https://github.com/Trial97) in [#2166](https://github.com/PrismLauncher/PrismLauncher/pull/2166)
```
Comment (DioEgizio) [inline src/news/release-8.4.md:45]:
```suggestion
- Update FTB App import instance path detection by [@Trial97](https://github.com/Trial97) in [#2166](https://github.com/PrismLauncher/PrismLauncher/pull/2166)
```

File: src/news/release-8.4.md [hunk 11]
```diff
@@ -0,0 +1,56 @@
...
+- Update imgur upload by [@Trial97](https://github.com/Trial97) in [#2499](https://github.com/PrismLauncher/PrismLauncher/pull/2499)
```
Comment (DioEgizio) [inline src/news/release-8.4.md:46]:
```suggestion
- Update Imgur upload by [@Trial97](https://github.com/Trial97) in [#2499](https://github.com/PrismLauncher/PrismLauncher/pull/2499)
```

File: src/news/release-8.4.md [hunk 12]
```diff
@@ -0,0 +1,56 @@
...
+- Use proxy style to force ActivateItemOnSingleClick off by [@TheKodeToad](https://github.com/TheKodeToad) in [#2348](https://github.com/PrismLauncher/PrismLauncher/pull/2348)
```
Comment (DioEgizio) [inline src/news/release-8.4.md:47]:
```suggestion
- Force single click to select UX by [@TheKodeToad](https://github.com/TheKodeToad) in [#2348](https://github.com/PrismLauncher/PrismLauncher/pull/2348)
```

File: src/news/release-8.4.md (hunk 8 above)
Comment (Trial97) [reply src/news/release-8.4.md:30]: I can't find the words to improve this, but any suggestion is welcome.
Is `Enhance the process of removing invalid characters from a path.` better?

**Review by DioEgizio — APPROVED** [review]

**Review by TayouVR — APPROVED** [review]

**Pull request merged**